### PR TITLE
Make mycryptobuilds use dev build

### PR DESCRIPTION
**Pull request by blurpesec**
[ch6397]

### Description
Makes a downloadable development build for use in mycryptobuilds.

### Changes:
- Created a new build for staging (using `NODE_ENV` = 'staging').
- Changed build destination directory `dist/download` to `dist/web`.
- Updated README to reflect new realities.
- Moved standalone production build to github ci and save it to artifacts.
- Moved storybook build to it's own workflow.

### CODE REVIEW

**Comment by FrederikBolding** [comment]
@blurpesec Do we not need `downloadable` for the standalone builds? I.e. for the people who want to download a production build of the website?

**Comment by blurpesec** [comment]
Turns out the difference between production build and downloadable was only environment variables. So I just removed them 🤷

**Comment by blurpesec** [comment]
> As discussed, change such that we still build the standalone

I added this build to the Github CI with the other builds :-)

File: electron-app/main/window.ts [hunk 1]
```diff
@@ -4,7 +4,8 @@ import MENU from './menu';
 import popupContextMenu from './contextMenu';
 import { APP_TITLE } from '../constants';
 
-const IS_DEV: boolean = process.env.NODE_ENV === 'development' || !process.env.NODE_ENV;
+const IS_DEV: boolean =
+  process.env.NODE_ENV === 'development' || !!process.env.IS_STAGING || !process.env.NODE_ENV;
```
Comment (vilsbole) [inline electron-app/main/window.ts:8]: 1. Staging is a separate environment and should not be confused with the `development` environment.
2. Prefer `process.env.NODE_ENV === 'staging'` for consistency

File: webpack_config/common.js [hunk 2]
```diff
@@ -9,7 +9,7 @@ const StyleLintPlugin = require('stylelint-webpack-plugin');
 const TsconfigPathsPlugin = require('tsconfig-paths-webpack-plugin');
 const config = require('./config');
 
-const IS_DEVELOPMENT = process.env.NODE_ENV !== 'production';
+const IS_DEVELOPMENT = process.env.NODE_ENV === 'development';
```
Comment (vilsbole) [inline webpack_config/common.js:12]: Use shared constant IS_DEV

File: webpack_config/common.js [hunk 3]
```diff
@@ -9,7 +9,7 @@ const StyleLintPlugin = require('stylelint-webpack-plugin');
...
 const IS_ELECTRON = !!process.env.BUILD_ELECTRON;
```
Comment (vilsbole) [inline webpack_config/common.js:13]: Replace with utils/platform

File: package.json [hunk 4]
```diff
@@ -229,9 +229,8 @@
   },
   "scripts": {
     "build": "cross-env NODE_ENV=production webpack --config webpack_config/production.js",
+    "build:staging": "cross-env NODE_ENV=staging webpack --config webpack_config/staging.js",
```
Comment (vilsbole) [inline package.json:232]: Staging should have the same build config as production.
Prefer
`build:staging": "cross-env NODE_ENV=staging webpack --config webpack_config/production.js`

File: src/index.tsx [hunk 5]
```diff
@@ -15,7 +15,7 @@ import Root from './Root';
 
 render(<Root />, document.getElementById('app'));
 
-if (process.env.NODE_ENV === 'production') {
+if (process.env.NODE_ENV === 'production' || !!process.env.IS_STAGING) {
```
Comment (vilsbole) [inline src/index.tsx:18]: Use constants

File: src/utils/environment.ts [hunk 6]
```diff
@@ -1,5 +1,3 @@
-export const IS_DEV: boolean = process.env.NODE_ENV === 'development';
-
-export const IS_PROD: boolean = process.env.NODE_ENV === 'production';
+export const IS_DEV: boolean = process.env.NODE_ENV === 'development' || !!process.env.IS_STAGING;
```
Comment (vilsbole) [inline src/utils/environment.ts:1]: IS_DEV is a seperate flag from IS_STAGING.

**Pull request merged**